lib/lynx.rb

```diff
@@ -34,6 +34,10 @@ def run
       Pipe::Run.new
     end
 
+    def popen(*args)
+      Pipe::Popen.new(*args)
```

Comment:
good catch.  updated.  there are no tests for this class but I modified the `p_open_test` to run through this method.